src/System.Diagnostics.Process/tests/Configurations.props

```diff
@@ -4,8 +4,9 @@
     <BuildConfigurations>
       netstandard-Windows_NT;
       netstandard-Unix;
+      netstandard-OSX;
```

Comment:
>  netstandard-OSX; [](start = 4, length = 18)

I think we can write the tests to work on different platforms (Linux and OSX) and we can pass different data to such tests according to the OS we are running on. We can use PlatformDetection to decide what set of data we can pass to the tests. doing that we don't have to add OSX configurations.